Upstream-Status: Pending

It fails to build crash for mips with error:

| mips-wrs-linux-gcc: error: unrecognized command line option '-m32'
| Makefile:291: recipe for target 'make_build_data' failed

So remove the unrecognized option '-m32' for mips.

Signed-off-by: Kai Kang <kai.kang@windriver.com>
---
diff --git a/configure.c b/configure.c
index cf1973b..71e97b3 100644
--- a/configure.c
+++ b/configure.c
@@ -151,7 +151,7 @@ void add_extra_lib(char *);
 #define TARGET_CFLAGS_PPC64_ON_X86_64  "TARGET_CFLAGS="
 #define TARGET_CFLAGS_MIPS            "TARGET_CFLAGS=-D_FILE_OFFSET_BITS=64"
 #define TARGET_CFLAGS_MIPS_ON_X86     "TARGET_CFLAGS=-D_FILE_OFFSET_BITS=64"
-#define TARGET_CFLAGS_MIPS_ON_X86_64  "TARGET_CFLAGS=-m32 -D_FILE_OFFSET_BITS=64"
+#define TARGET_CFLAGS_MIPS_ON_X86_64  "TARGET_CFLAGS=-D_FILE_OFFSET_BITS=64"
 #define TARGET_CFLAGS_SPARC64         "TARGET_CFLAGS="
 
 #define GDB_TARGET_DEFAULT        "GDB_CONF_FLAGS="
@@ -162,7 +162,7 @@ void add_extra_lib(char *);
 #define GDB_TARGET_ARM64_ON_X86_64  "GDB_CONF_FLAGS=--target=aarch64-elf-linux"   /* TBD */
 #define GDB_TARGET_PPC64_ON_X86_64  "GDB_CONF_FLAGS=--target=powerpc64le-unknown-linux-gnu"
 #define GDB_TARGET_MIPS_ON_X86     "GDB_CONF_FLAGS=--target=mipsel-elf-linux"
-#define GDB_TARGET_MIPS_ON_X86_64  "GDB_CONF_FLAGS=--target=mipsel-elf-linux CFLAGS=-m32"
+#define GDB_TARGET_MIPS_ON_X86_64  "GDB_CONF_FLAGS=--target=mipsel-elf-linux"
      
 /*
  *  The original plan was to allow the use of a particular version
